build.sh

```diff
@@ -8,7 +8,16 @@ __nugetconfig=$__sourceroot/NuGet.Config
 __msbuildpackageid="Microsoft.Build.Mono.Debug"
 __msbuildpackageversion="14.1.0.0-prerelease"
 __msbuildpath=$__packageroot/$__msbuildpackageid.$__msbuildpackageversion/lib/MSBuild.exe
-__referenceassemblyroot=/usr/lib/mono/xbuild-frameworks
+
+if [ $(uname) == "Linux" ]; then
+    __monoroot=/usr
+else
+    __monoroot=/Library/Frameworks/Mono.framework/Versions/4.0.0
```

Comment:
Sorry, when I was searching for it I didn't see the portable folder - love .Net being hidden. :)